### ISSUE TITLE
Drum panel not accessible by keyboard

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#24741]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. Create a score for "large drum kit"
2. Enter note input mode
3. Attempt to move keyboard focus to the drum palette using Tab or F6

Result: it is skipped in the tab order, so there is no way to use the palette by keyboard at all.  There is no way to transfer focus by clicking either (not that there *should* be - normally clicking a palette should not change focus), so I can't say if the palette *would* be keyboard-accessible once you get focus there.  I'm guessing it would be and the problem is just getting focus there.

### Supporting files, videos and screenshots

[Screen recording 2024-09-15 2.33.08 PM.webm](https://github.com/user-attachments/assets/51b7fd9d-4f96-4469-b428-9e1b01114159)

### What is the latest version of MuseScore Studio where this issue is present?

OS: Debian GNU/Linux 12 (bookworm), Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242590506, revision: github-musescore-musescore-3130f97

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Debian 12, also confirmed on Windows 11

### Additional context

This is not a recent regression, I can reproduce going back to 4.2.  Which is surprising to me, as this is a pretty huge accessibility issue that apparently escaped notice until a user reported it on a mailing list recently.

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by bkunda** [comment]
In our plans for the replacement to this drum panel, accessibility has been given a lot of focus. 
So I can say this issue will be fixed, but not for the present panel. Hence, we'll close this for now.
